dependencies.json

```diff
@@ -150,11 +150,11 @@
         }
       },
       "fontAwesome": {
-        "version": "5.9.0",
+        "version": "6.4.2",
```

Comment:
(Maybe I'm missing something) It looks like github icon exist from version 2.0.0 ([ref](https://fontawesome.com/icons/github?f=brands&s=solid)). If yes, is there any specific reason for this updgrade?

(Its better to avoid upgrade without going through proper testing and ensutring it will not affect any other part of the codebase.)

Reply:
Hi,I have reverted the upgrade,so now it is the previous one only(ie. 5.9.0)
Please check the commit named:**Changes after review.**

Reply:
@shri3016 I'm assuming you have tested the UI with icon from v5.9.0. Would you mind sharing teh screenshot presenting the icons? (This is just to update the PR with screenshot of UI with the latest commit.)

Reply:
@shri3016 Can you please help resolve this thread before we merge this PR?

Reply:
Hello @DubeySandeep,Sorry I missed this comment.Here is the screenshot.
![gitlogoss](https://github.com/oppia/oppia/assets/73349138/188b95cc-24d1-4f0b-abe7-b9531e9d871b)